internal/compose/compose.go

```diff
@@ -199,20 +198,13 @@ func NewProject(name string, paths ...string) (*Project, error) {
 	// Passing a nil context here because we are on initialization.
 	ver, err := c.dockerComposeVersion(context.Background())
 	if err != nil {
-		logger.Errorf("Unable to determine Docker Compose version: %v. Defaulting to 1.x", err)
-		c.dockerComposeV1 = true
+		logger.Errorf("Unable to determine Docker Compose version: %v", err)
 		return &c, nil
 	}
-
-	versionMessage := fmt.Sprintf("Determined Docker Compose version: %v", ver)
-	if ver.Major() == 1 {
```

Comment:
Should we keep this check and raise an error if versions 1.* are used ?

Reply:
Kept, good point.